README.md

```diff
@@ -1,17 +1,25 @@
 # canopas-blog-components
```

Comment:
Please remove the `canopas` prefix. As we are going to remove that from library name